test/e2e/basic_workflow_test.go

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
Add `// +build e2e` to all your E2E tests, otherwise the presubmit job will try to run them as unit tests.

Reply:
But now they don’t execute locally with `test/e2e-tests-locals.sh`. I am assuming I need to setup something there to make that work. Searching google...

Reply:
Got it... adding `—tags ‘e2e’` does the trick

Reply:
Yep :)